CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/)
 
 ## [unreleased] (Added 🚀 | Changed | Removed 🗑 | Fixed 🐞 | Chore 👨‍💻 👩‍💻)
 
+### Added 🚀
+
+-   Add new Analysis docker container, together with a docker-compose file to quickly get started [#3057](https://github.com/MaibornWolff/codecharta/pull/3057)
```

Comment:
Would you please mention that the docker-compose is also providing a SonarQube instance?